src/scale_audio_rating.py

```diff
@@ -248,6 +249,20 @@ async def prepare_metadata_per_task(cfg, clips, gold, trapping, output_dir):
 
     return metadata_lst
 
+async def create_batch(scale_api_key,project_name,clips_file_name):
+    # create scale batch using the clips file name
+    batch_name = clips_file_name.clip('.')[0] + "_" + date.today()

```

Comment:
I believe `project_name` here refers to MSFT project, which is equivalent to the batch name. The Scale project is collected through `cfg.get("CommonAccountKeys", 'ScaleAccountName')` below.

Reply:
in that case, should we use that project name as the batch name instead of the `clips_file_name`?

Reply:
Yeah, the batch name should be the same as `args.project`